front/src/modules/trainschedule/helpers/__tests__/pacedTrain.spec.ts

```diff
@@ -35,3 +38,109 @@ describe('getOccurrencesNb', () => {
     ).toThrow('Interval cannot be 0');
   });
 });
+
+describe('formatPacedTrainWithOccurenceDetails', () => {
+  const pacedTrain: PacedTrainWithDetails = {
+    id: 'paced_123' as PacedTrainId,
+    name: '8608',
+    labels: ['Paced-Train-Tag-1', 'SS-NS', 'Valid'],
+    startTime: new Date('2024-10-15T03:00:00Z'),
+    path: [
+      { id: 'id227', deleted: false, uic: 6, secondary_code: 'BV' },
+      { id: 'id228', deleted: false, uic: 5, secondary_code: 'BV' },
+    ],
+    schedule: [
+      {
+        at: 'id228',
+        arrival: null,
+        stop_for: 'P0D',
+        reception_signal: 'OPEN',
+        locked: false,
+      },
+    ],
+    margins: { boundaries: [], values: ['0%'] },
+    options: {
+      use_electrical_profiles: false,
+    },
+    comfort: 'STANDARD',
+    constraint_distribution: 'MARECO',
+    speedLimitTag: 'MA100',
+    paced: { timeWindow: Duration.parse('PT2H'), interval: Duration.parse('PT1H') },
+    arrivalTime: new Date('2024-10-15T04:00:00Z'),
+    duration: Duration.parse('PT1H'),
+    stopsCount: 0,
+    pathLength: '200',
+    mechanicalEnergyConsumed: 5000,
+    isValid: true,
+    exceptions: [],
+  };
+
+  it('should properly update a standard property', () => {
+    const exception: PacedTrainException = {
+      key: '123123',
+      train_name: { value: '8608 updated' },
+    };
+    const updatedPacedTrain = formatPacedTrainWithOccurenceDetails(pacedTrain, exception);
+    expect(updatedPacedTrain).toEqual({
+      ...pacedTrain,
+      name: '8608 updated',
+    });
```

Comment:
I think the id should also change no ?
```suggestion
    expect(updatedPacedTrain).toEqual({
      ...pacedTrain,
      id: 'exception_123_123123'
      name: '8608 updated',
    });
```

Reply:
Hmm I don't think so. Here we are displaying the paced train with the exceptions changes and we will still want to send the paced train infos (including its id) to update it and its exceptions

Reply:
Ah yes you're right ! :+1: